apps/settings/js/panels/root/panel.js

```diff
@@ -158,3 +133,29 @@ define(function(require) {
     });
   };
 });
+/**
+ * Update the sim and NFC related items
+   based on mozMobileConnections and Device.
+ */
+
+var updateSimItemsAndNfc = function rph_refrehsSimItems() {
```

Comment:
1.Actually all other js code for Network and Connectivity is in panel.js by mozilla so we place the update telephone items in same js .
2.Still NFC is hide/show on basis of either device supports NFC or not only it will be called in same function.
3. We didn' t remove SimCardManager condition because we are not sure wheather mozilla supports dual sim non-touch phones on not if it supports.
4.In Previous case all items loads on same root page but now privacy and network items loads  different pages so no need to load complete js for other category .